src/twisted/test/test_failure.py

```diff
@@ -535,10 +535,10 @@ def test_repr(self):
 
 class BrokenStr(Exception):
     """
-    An exception class the instances of which cannot be presented as strings via
-    C{str}.
+    An exception class the instances of which cannot be presented as strings
+    via C{str}.
```

Comment:
Should be L{str} in doc-string

Reply:
I added a commit to fix this, but maybe we should do a dedicated PR to fix links to builtins, because I think this is a problem that happens a lot.